app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
+  },
+  "failed": {
+    "message": "Fallo"
+  },
+  "fiat": {
+    "message": "FIAT",
+    "description": "Exchange type"
+  },
+  "fileImportFail": {
+    "message": "Importar archivo no funciona? Click Aqui!",
+    "description": "Helps user import their account from a JSON file"
+  },
+  "from": {
+    "message": "De:"
+  },
+  "fromShapeShift": {
+    "message": "De ShapeShift"
+  },
+  "gas": {
+    "message": "Gas",
+    "description": "Short indication of gas cost"
+  },
+  "gasFee": {
+    "message": "Comision de gas"
+  },
+  "gasLimit": {
+    "message": "Limit de gas"
+  },
+  "gasLimitCalculation": {
+    "message": "Calculamos el límite de gas sugerido en función de las tasas de éxito de la red."
+  },
+  "gasLimitRequired": {
+    "message": "Limite de Gas requerido"
+  },
+  "gasLimitTooLow": {
+    "message": "El límite de gas debe ser de al menos 21000"
+  },
+  "gasPrice": {
+    "message": "Precio del Gas (GWEI)"
+  },
+  "gasPriceCalculation": {
+    "message": "Calculamos los precios sugeridos del gas en función de las tasas de éxito de la red."
+  },
+  "gasPriceRequired": {
+    "message": "Precio del GAS requerido"
+  },
+  "getEther": {
+    "message": "Conseguir Ether"
+  },
+  "getEtherFromFaucet": {
+    "message": "Obtenga Ether desde un foucet por $1",
+    "description": "Displays network name for Ether faucet"
+  },
+  "greaterThanMin": {
+    "message": "Debe ser mayor o igual a $1.",
+    "description": "helper for inputting hex as decimal input"
+  },
+  "here": {
+    "message": "Aqui",
+    "description": "as in -click here- for more information (goes with troubleTokenBalances)"
+  },
+  "hide": {
+    "message": "Ocultar"
+  },
+  "hideToken": {
+    "message": "Ocultar Token"
+  },
+  "hideTokenPrompt": {
+    "message": "Ocultar Token?"
+  },
+  "howToDeposit": {
+    "message": "Como te gustaria depositar Ether?"
+  },
+  "import": {
+    "message": "Importar",
+    "description": "Button to import an account from a selected file"
+  },
+  "importAccount": {
+    "message": "Importar Cuenta"
+  },
+  "importAnAccount": {
+    "message": "Importar una cuenta"
+  },
+  "importDen": {
+    "message": "Importar DEN existente"
+  },
+  "imported": {
+    "message": "Importado",
+    "description": "status showing that an account has been fully loaded into the keyring"
+  },
+  "infoHelp": {
+    "message": "Informacion y Ayuda"
+  },
+  "invalidAddress": {
+    "message": "Direccion invalida"
+  },
+  "invalidGasParams": {
+    "message": "Parametros de Gas Invalidos"
+  },
+  "invalidInput": {
+    "message": "Entrada invalida."
+  },
+  "invalidRequest": {
+    "message": "Peticion invalida"
+  },
+  "invalidAddressRecipient": {
+    "message": "Direccion del recipiente invalida"
+  },
+  "fromToSame": {
+    "message": "La direccion de origen y destino no pueden ser la misma"
+  },
+  "jsonFile": {
+    "message": "Archivo JSON",
+    "description": "format for importing an account"
+  },
+  "jsonFail": {
```

Comment:
https://github.com/MetaMask/metamask-extension/blob/master/app/_locales/en/messages.json not inin here @danfinlay